javascript/javascript-maps.md

```diff
@@ -41,6 +41,8 @@ Map object provided by ES6. Map is a collection of elements where each element i
 
 ## Lesson
 
+* **Reference**: [Map, Set](https://javascript.info/map-set-weakmap-weakset)
```

Comment:
Mention website or author instead of Map, Set, minimum.  If you want to include the title, make sure its both the title and author.  This applies for the other outline in this PR, too.

Reply:
Did you need  to cite the motivation section too?

Reply:
No need for motivation section. It contains general terms.

Reply:
My search says that your motivations section is from geekforgeeks.

Reply:
I studied these topics from many different sources and this file is completely from my notes i made during visiting many sites. I guess thats why some of words matches.

Reply:
If you used these sources in your notes, and you are using the exact words, you should still cite them:
https://smallseotools.com/view-report/d296428abc3c676592075772854244e6/

Reply:
Done